WalletWasabi.Documentation/BackendDeployment.md

```diff
@@ -155,8 +155,6 @@ testnet=[0/1]
 
 [main/test].daemon=1
 [main/test].server=1
-[main/test].rpcuser=bitcoinuser
-[main/test].rpcpassword=password
```

Comment:
If you delete it how does Bitcoin Core know?

Reply:
> When no -rpcpassword is specified, the daemon now uses a special ‘cookie’ file for authentication. 

In theory no. Did you test it?

Reply:
https://github.com/bitcoin/bitcoin/blob/master/src/httprpc.cpp#L217

Yes, I tested it.